pkg/cosign/image_signer_test.go

```diff
@@ -93,16 +93,16 @@ func testImageSigner(t *testing.T, when spec.G, it spec.S) {
 				// Override secretLocation for test
 				secretLocation = createCosignKeyFiles(t)
 
-				secretKey1 = path.Join(secretLocation, "secret-name-1", cosignutil.SecretDataCosignKey)
-				publicKey1 = path.Join(secretLocation, "secret-name-1", cosignutil.SecretDataCosignPublicKey)
-				publicKey2 = path.Join(secretLocation, "secret-name-2", cosignutil.SecretDataCosignPublicKey)
-				passwordFile1 = path.Join(secretLocation, "secret-name-1", cosignutil.SecretDataCosignPassword)
-				passwordFile2 = path.Join(secretLocation, "secret-name-2", cosignutil.SecretDataCosignPassword)
+				secretKey1 = path.Join(secretLocation, "secret-name-1", secret.CosignSecretPrivateKey)
+				publicKey1 = path.Join(secretLocation, "secret-name-1", secret.CosignSecretPublicKey)
+				publicKey2 = path.Join(secretLocation, "secret-name-2", secret.CosignSecretPublicKey)
+				passwordFile1 = path.Join(secretLocation, "secret-name-1", secret.CosignSecretPassword)
+				passwordFile2 = path.Join(secretLocation, "secret-name-2", secret.CosignSecretPassword)
 
 				report = createReportToml(t, expectedImageName, imageDigest)
 
-				os.Unsetenv(cosignutil.CosignRepositoryEnv)
-				os.Unsetenv(cosignutil.CosignDockerMediaTypesEnv)
+				os.Unsetenv(CosignRepositoryEnv)
```

Comment:
Do we have any guarantees about the order of the tests?  AFAIK unsetting these env vars here have an impact on tests that run subsequently.  Do we want to change the environment for subsequent tests?